storage_controller/src/safekeeper.rs

```diff
@@ -18,19 +18,22 @@ pub struct Safekeeper {
     cancel: CancellationToken,
     listen_http_addr: String,
     listen_http_port: u16,
+    scheduling_policy: SkSchedulingPolicy,
     id: NodeId,
     availability: SafekeeperState,
 }
 
 impl Safekeeper {
     pub(crate) fn from_persistence(skp: SafekeeperPersistence, cancel: CancellationToken) -> Self {
+        let scheduling_policy = SkSchedulingPolicy::from_str(&skp.scheduling_policy).unwrap();
```

Comment:
FTR this will make the safekeeper crash if it tries to load an invalid scheduling policy. But this is impossible to reach without manual SQL queries, because we have a default value for that scheduling policy.